trview.app/Resources/Files/type_names.txt

```diff
@@ -2991,8 +2991,8 @@
                 "categories": [ "Trap" ]
             }, {
                 "id": 135,
-                "name": "Flame emitter",
-                "categories": [ "Trap" ]
+                "name": "Chain",
+                "categories": [ "Scenery" ]
```

Comment:
It's been a long time since I played TR4, but does it deal damage? Maybe it is still a trap.

Reply:
I don't know either! Will ask around or maybe even play the game